clients/unshaded/src/main/java/tachyon/client/file/FileSystemMasterClient.java

```diff
@@ -77,6 +83,144 @@ protected void afterConnect() throws IOException {
   }
 
   /**
+   * Creates a new file.
+   *
+   * @param path the file path
+   * @param options method options
+   * @return the file id referencing the newly created file
+   * @throws IOException if an I/O error occurs
+   * @throws TachyonException if a Tachyon error occurs
+   */
+  public synchronized long create(final String path, final CreateOptions options)
```

Comment:
ah, this is probably temporary ... maybe adding a TODO to remove this would be helpful